backend/src/v2/driver/driver.go

```diff
@@ -475,6 +475,12 @@ func extendPodSpecPatch(
 		podSpec.Containers[0].VolumeMounts = append(podSpec.Containers[0].VolumeMounts, volumeMounts...)
 	}
 
+	// Get image pull policy
+	if kubernetesExecutorConfig.GetImagePullPolicy() != "" {
```

Comment:
Maybe we can also set kubernetesExecutorConfig.GetImagePullPolicy() as a variable, then check whether the value is one of ['Always', 'Never', 'IfNotPresent']?

There could be a case where someone can compose the IR without the SDK, so we also wants to check the value is valid before passing into the pod spec. If the value is not valid, we should raise an error instead.

Reply:
@Tomcli Done. Thanks